### PR TITLE
Add task descriptions to README

### PR DESCRIPTION
**Pull request by TadCordle**
#204

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,21 +59,24 @@ buildscript {
...
 
 # Supported tasks
-- appengineStage
-- appengineDeploy
-- appengineDeployCron
-- appengineDeployDispatch
-- appengineDeployDos
-- appengineDeployIndex
-- appengineDeployQueue
+| Goal                       | Description
```
Comment (etanshaul) [inline README.md]: nit: are these called goals or tasks (I thought the gradle terminology was tasks)? @loosebazooka

Comment (TadCordle) [reply README.md]: Yeah, I think they're tasks. Was referencing the maven plugin readme and forgot to change this.

File: README.md [hunk 2]
```diff
@@ -59,21 +59,24 @@ buildscript {
...
+| appengineShowConfiguration | Shows the current App Engine plugin configuration properties.
+| appengineStage             | Generates an application directory for deployment.
+| appengineDeploy            | Stages and deploys an application to App Engine.
+| appengineDeployCron        | Deploys cron configuration (cron.yaml).
+| appengineDeployDispatch    | Deploys dispatch configuration (dispatch.yaml).
+| appengineDeployDos         | Deploys dos configuration (dos.yaml).
+| appengineDeployIndex       | Deploys datastore index configuration (index.yaml).
+| appengineDeployQueue       | Deploys queue configuration (queue.yaml).
 
 Dev App Server goals for standard environment apps only:
```
Comment (etanshaul) [inline README.md]: goals -> tasks

**Comment by loosebazooka** [comment]
There's a user guide file that we should probably merge and update with this.

**Pull request merged**